clr-installer/main.go

```diff
@@ -221,7 +222,13 @@ func execute(options args.Args) error {
 		if cf, err = conf.LookupDefaultConfig(); err != nil {
 			return err
 		}
+	} else if network.IsValidURI(options.ConfigFile, options.AllowInsecureHTTP) {
+		if cf, err = network.FetchRemoteConfigFile(options.ConfigFile); err != nil {
+			return err
+		}
+		options.CfDownloaded = true
 	}
```

Comment:
We need to add an else here since we do not have a valid configuration file.
```suggestion
	} else {
	    return errors.Errorf("No valid configuration file")	    
	}
```